code/script.js

```diff
@@ -4,16 +4,117 @@
 // Your code goes here
 alert(
   `Welcome to our Javascript Pizzeria. Ready to Start? - Click 'OK' to begin.`
-)
+);
+// Ask the user for their name and store the input in a variable
+const userName = prompt(`Please enter your name:`);
 
+// Display a greeting using the user's name
+if (userName) {
+  alert(`Hello, ${userName} ! Welcome to our Javascript Pizzeria. 🍕`);
+} else {
+  alert(`No name entered.☹️`);
+};
 // Step 2 - Food choice
-// Your code goes here
+const foodChoice = prompt(`What food would you like to order? 
+ Enter a number:
+ 1 - Pizza
+ 2 - Pasta
+ 3 - Salad`);
+
+let foodType;
+
+if (foodChoice === `1`) {
+  foodType = `Pizza`;
+  alert(`You chosen  pizza.`);
```

Comment:
Not really related to the actual JS code, but messages should be consistent throughout. For example, you use "You chosen" in some messages and "You've ordered" in others. Also a few spaces too many.